tools/codesigningtool/codesigningtool.py

```diff
@@ -103,8 +124,17 @@ def main(argv):
         print("ERROR: Unable to find an identity on the system matching the "\
             "ones in %s" % args.mobileprovision, file=sys.stderr)
         return 1
-    os.execve(args.codesign, [args.codesign, "-v", "--sign", identity] +
-        codesign_args, os.environ)
+    stdout, stderr = _check_output(
+        [args.codesign, "-v", "--sign", identity] + codesign_args,
+    )
+    if stdout:
+        filtered_stdout = _filter_codesign_output(stdout)
+        if filtered_stdout:
+            print(filtered_stdout)
+    if stderr:
+        filtered_stderr = _filter_codesign_output(stderr)
```

Comment:
do any of the benign messages go to stderr? Just wondering if stderr should be unfiltered.

Reply:
yup, the annoying ones go through `stderr`